test/engine-luatest/gh_6986_c_constraint_test.lua

```diff
@@ -0,0 +1,33 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'test_c_constraint'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+g.test_c_constraint = function()
+    g.server:exec(function()
+        local build_path = os.getenv("BUILDDIR")
+        package.cpath = build_path..'/test/engine-luatest/?.so;'..
+                        build_path..'/test/engine-luatest/?.dylib;'..
+                        package.cpath
+        local t = require('luatest')
+        local func = {language = 'C', is_deterministic = true}
+        box.schema.func.create('gh_6986.get_check', func);
```

Comment:
Engine tests are supposed to be multi-engine. This one isn't.

Reply:
Thank you for the review! Fixed.